src/types/Health.ts

```diff
@@ -25,7 +25,7 @@ export interface EnvoyRatio {
   total: number;
 }
 
-export interface DeploymentStatus {
+export interface WorkloadStatus {
```

Comment:
I wonder if "ReplicasStatus" would be a more accurate name, after all. This health component aims to show how many replcias are running versus how many are desired. What do you think?

Reply:
I remember you commented same thing in the backend PR and we finish to leave as WorkloadStatus. I would maintain backend -> frontend mapping for consistency, too.